app/scripts/formentry/controllers/formentry.controller.js

```diff
@@ -856,6 +856,18 @@ jshint -W098, -W003, -W068, -W004, -W033, -W030, -W117, -W069, -W106
 
             //first stage of submitting is to save new obs
 
+            //TODO - APTS-146 fix
+
+            if(lastPayload && lastPayload.obs && Array.isArray(lastPayload.obs)) {
```

Comment:
The payload could be recursive in nature. There are multiple fields affected by this bug. Use a recursive function to find all voided obs and delete their value field